android/alarmclock/res/values-ua/strings.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Translation by av.n...@gmail.com -->
+<resources>
+  <string name="app_name">Alarm Klock</string>
+  <string name="delete_all">Видалити всі будильники</string>
+  <string name="label">Мітка</string>
+  <string name="repeat">Повтор</string>
+  <string name="alarm_tone">Мелодія будильника</string>
+  <string name="fade_description">Від %1$d%% до %2$d%% за %3$d секунд.</string>
+  <string name="ok">ОК</string>
+  <string name="cancel">Відмінити</string>
+  <string name="delete">Видалити</string>
+  <string name="snooze">ПОДРІМАТИ</string>
+  <string name="dismiss">ВИМКНУТИ</string>
+  <string name="plus_five">+5</string>
+  <string name="minus_five">-5</string>
+  <string name="day">%d день</string>
+  <string name="days">%d днів</string>
+  <string name="hour">%d година</string>
+  <string name="hours">%d годин</string>
+  <string name="minute">%d хвилина</string>
+  <string name="minutes">%d хвилин</string>
+  <string name="am">AM</string>
+  <string name="pm">PM</string>
+  <string name="dow_sun_short">Нд</string>
+  <string name="dow_mon_short">Пн</string>
+  <string name="dow_tue_short">Вт</string>
+  <string name="dow_wed_short">Ср</string>
+  <string name="dow_thu_short">Чт</string>
+  <string name="dow_fri_short">Пт</string>
+  <string name="dow_sat_short">Сб</string>
+  <string name="dow_sun">Неділя</string>
+  <string name="dow_mon">Понеділок</string>
+  <string name="dow_tue">Вівторок</string>
+  <string name="dow_wed">Середа</string>
+  <string name="dow_thu">Четвер</string>
+  <string name="dow_fri">П'ятниця</string>
+  <string name="dow_sat">Субота</string>
+  <string name="no_repeats">Без повтора</string>
+  <string name="everyday">Кожен день</string>
+  <string name="weekdays">Робочі дні</string>
+  <string name="weekends">Вихідні</string>
+  <string name="internal">Встроєні</string>
```

Comment:
It's not "clean" ukrainian, but mixed with russian.

Reply:
It is "clean" Ukrainian unlike in of your translations